core/src/main/kotlin/com/malinskiy/marathon/report/logs/LogReport.kt

```diff
@@ -0,0 +1,21 @@
+package com.malinskiy.marathon.report.logs
+
+import java.io.File
+
+class LogReport(
+    val batches: Map<String, BatchLogs>
+)
+
+class BatchLogs(
+    val tests: Map<LogTest, Log>,
+    val log: Log
+)
+
+class Log(
+    val file: File,
+    val events: List<LogEvent>
```

Comment:
it's currently not used, the plan is to parse errors (such as "SIGSEGV" crashes) from the logcat and add them here, so then we can add the failure message to the test failure result